detox/ios/Detox/DetoxAppDelegateProxy.m

```diff
@@ -335,7 +335,7 @@ - (void)applicationWillEnterForeground:(UIApplication *)application
 
 - (BOOL)touchVisualizerWindowShouldAlwaysShowFingertip:(COSTouchVisualizerWindow *)window
 {
-	return YES;
+	return ![[NSUserDefaults standardUserDefaults] objectForKey:@"detoxDisableTouchIndicators"];
```

Comment:
Let's not read from the user defaults for every touch.

Please take my changes from https://github.com/wix/detox/pull/751/files